src/components/utils/dialog-utils.js

```diff
@@ -34,3 +38,20 @@ export const toFloatOrNullValue = (value) => {
     const tmp = value?.replace(',', '.') || '';
     return parseFloat(tmp) || null;
 };
+
+export const useCountriesListCB = () => {
+    const [languageLocal] = useParameterState(PARAM_LANGUAGE);
+    return useCallback(() => {
+        try {
+            return require('localized-countries')(
+                require('localized-countries/data/' +
+                    getComputedLanguage(languageLocal).substr(0, 2))
+            );
+        } catch (error) {
+            // fallback to English if no localized list is found
+            return require('localized-countries')(
+                require('localized-countries/data/en')
+            );
+        }
+    }, [languageLocal]);
+};
```

Comment:
In my opinion, can try to use the hook in gridstudy-app : src/components/utils/localized-countries-hook.js
We can duplicate this hook in gridexplore-app. Eventually, we have already a POC to merge between two apps in the next year

Reply:
I don't think duplicating the code is good in the current context as we might merge the gridstudy and gridexplore soon and duplicating this code might lead to divergence. We'll update this when the merge is done.

Reply:
OK